src/node.cpp

```diff
@@ -1343,6 +1428,18 @@ void LocalNode::setnameparent(LocalNode* newparent, string* newlocalpath)
     }
 }
 
+void LocalNode::reactToNodeChange(const bool nodeDeleted)
+{
+    if (!sync->getConfig().isDownSync())
```

Comment:
I suggest renaming isDownSync() to syncsDown() or similar.  isDownSync sounds like it's the one-way sync in that direction (excluding two-way), however it tests whether the sync moves changes in that direction (so true for one-way down sync and for two-way sync)

Reply:
syncsDown is problematic cuz it sounds like "the sync is down" in like not working. We also have `syncsup` of `MegaClient` so using `syncsUp` is problematic too.

Reply:
Open to other name suggestions tho

Reply:
how about doesSyncUp ?  syncsToCloud / syncsToLocal  maybe

Reply:
The latter sounds. Will change to that.